deployment/Dockerfile

```diff
@@ -1,8 +1,13 @@
 # Stage 1: Build
-FROM lukemathwalker/cargo-chef:latest-rust-1.69.0 AS chef
+FROM --platform=$BUILDPLATFORM tonistiigi/xx AS xx
```

Comment:
@Voxelot Would've taken me forever to find out about this. How did you come across this?